CONTRIBUTING.md

```diff
@@ -0,0 +1,76 @@
+This page contains instructions on how to propose and implement feature changes to Presto.
+
+# The Request for Comments
+
+## Overview of the process
+
+```mermaid
+graph TD;
+    id1["Create an RFC"]-->id2["Get feedback on the RFC"];
+    id2["Get feedback on the RFC"]-->id3["RFC is closed"];
+    id2["Get feedback on the RFC"]-->id4["RFC is merged"];
+    id4["RFC is merged"]-->id5["Create a Github issue linking the RFC"];
+    id5["Create a Github issue linking the RFC"]-->id6["Create PRs implementing the RFC (link to the issue)"];
+```
+
+## Step 1: Create an RFC
+RFCs are located in their own repository.
+
+To create one:
+
+1. Fork the https://github.com/prestodb/rfcs repository
+2. Copy the template file `RFC-0000-template.md` to `RFC-00xx-your-feature.md` and fill it out with your proposal.
+   The template is a guideline, feel free to add sections as appropriate
+3. You may also have the template simply link to another editor, like a Google Docs file, but please ensure that the
```

Comment:
I am a bit skeptical about giving this option/instruction. Even if google doc file is public, there is no guarantee it will be available after, say 1 year or so.

Reply:
We can say that before merge, it must be put into the main MD file.  This is just to reduce process and overhead to get the initial design out, but I can remove it.

Reply:
Yes, before merge it must be put into the main MD file. Then no issues.

Reply:
But I am bit leaning towards removing it, because ultimately the writeup has to be there in MD file only. We can wait to hear from others as well.

Reply:
I too prefer using the md file and PR for comments. It will provide consistency.